zokrates_js/Dockerfile.env

```diff
@@ -0,0 +1,22 @@
+FROM node:10
+SHELL ["/bin/bash", "-c"]
+
+ENV RUSTUP_HOME=/usr/local/rustup \
+    CARGO_HOME=/usr/local/cargo \
+    PATH=/usr/local/cargo/bin:$PATH
+
+RUN set -eux; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        gcc \
+        libc6-dev \
+        ; \
+    \
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --no-modify-path --default-toolchain nightly; \
```

Comment:
would it make sense for the Dockerfile to have the nightly version as a parameter somewhere so we can control that? Right now it gets the latest nightly?

Reply:
Yeah good point, I added build argument for specifying rust version (if argument is not specified it defaults to nightly)